src/Sylius/Bundle/UiBundle/DependencyInjection/SyliusUiExtension.php

```diff
@@ -17,15 +17,46 @@
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\DependencyInjection\Loader\XmlFileLoader;
 use Symfony\Component\HttpKernel\DependencyInjection\Extension;
+use Zend\Stdlib\SplPriorityQueue;
 
 final class SyliusUiExtension extends Extension
 {
-    /**
-     * {@inheritdoc}
-     */
     public function load(array $config, ContainerBuilder $container): void
     {
+        $config = $this->processConfiguration($this->getConfiguration([], $container), $config);
         $loader = new XmlFileLoader($container, new FileLocator(__DIR__ . '/../Resources/config'));
+
         $loader->load('services.xml');
+
+        $this->loadEvents($config['events'], $container);
+    }
+
+    /**
+     * @psalm-param array<string, array{blocks: array<string, array{template: string, priority: int, enabled: bool}>}> $eventsConfig
+     */
+    private function loadEvents(array $eventsConfig, ContainerBuilder $container): void
+    {
+        $templateEventExtensionDefinition = $container->getDefinition('sylius.twig.extension.template_event');
+
+        $blocksForEvents = [];
+        foreach ($eventsConfig as $eventName => $eventConfiguration) {
+            $blocksPriorityQueue = new SplPriorityQueue();
+
+            foreach ($eventConfiguration['blocks'] as $blockName => $details) {
+                if ($details['enabled'] === false) {
```

Comment:
Can we pass any value in the `enabled` key? Even some string and it won't be parsed to bool?

Reply:
Enabled is `booleanNode`, so it shouldn't be possible.

Reply:
So maybe `if(!$details['enabled'])` is enought? 😄